src/project/SidebarView.js

```diff
@@ -90,6 +89,8 @@ define(function (require, exports, module) {
      * Show the sidebar.
      */
     function show() {
+	$sidebar.width(MinimumSidebarWidth); 
```

Comment:
Not quite right.  You should check the current width, and only change it if it's less than the minimum:

```js
if($sidebar.width() < minimumSidebarWidthPX) {
    $sidebar.width(minimumSidebarWidthPX);
}
```